walletnode/api/design/artworks.go

```diff
@@ -307,3 +340,190 @@ var RegisterTaskPayload = Type("RegisterTaskPayload", func() {
 	})
 	Required("taskId")
 })
+
+// SearchArtworkParams are query params to searchArtwork request
+var SearchArtworkParams = func() {
+	Attribute("artist", String, func() {
+		Description("Artist PastelID or special value; mine")
+		MaxLength(256)
+	})
+	Attribute("limit", Int, func() {
+		Description("Number of results to be return")
+		Minimum(10)
+		Default(10)
+		Example(10)
+	})
+
+	Attribute("artist_name", String, func() {
+		Description("search for matches in ticket's field - artist_name")
```

Comment:
```suggestion
		Description("Name of the artist")
```
The aim of the Description is to describe the field. This is not endpoint. Please fix it everywhere

Reply:
but these are filter query params , still?